lib/mtif/posts.rb

```diff
@@ -125,20 +125,26 @@ def mtif_value(value)
       value.kind_of?(Time) ? value.strftime(DATE_FORMAT) : value
     end
 
-    def convert_to_native_type(raw_value)
+    def convert_to_native_type(raw_value, key)
+      if MULTILINE_KEYS.include?(key)
```

Comment:
I don't love having to pass the key in here, and I see why you'd need to. I think I want to refactor how keys are defined and typed to make those relationships more explicit (e.g. make the key constants hashes of `keyname => type`, or somesuch).